src/xref_runner.erl

```diff
@@ -219,7 +219,9 @@ find_function_source(M, F, A, Bin) ->
   {ok, {M, [{abstract_code, {raw_abstract_v1, Code}}]}} = AbstractCode,
 
   %% Extract the original source filename from the abstract code
-  [{attribute, 1, file, {Source, _}} | _] = Code,
+  [{_, _, _, {Source, _}} | _] = [E || E <- Code,
```

Comment:
It would be better to use a proper generator instead of 2 filters, like this:

```erlang
[Source|_] = [S || {attribute, _, file, {Source, _}} <- Code]
```
```